server/etcdmain/help.go

```diff
@@ -293,7 +293,7 @@ Experimental feature:
     Duration of periodical watch progress notification.
   --experimental-warning-apply-duration '100ms'
     Warning is generated if requests take more than this duration.
-  --experimental-txn-mode-write-with-shared-buffer 'true'
+  --experimental-txn-mode-write-with-shared-buffer 'true'. It's deprecated, and will be decommissioned in v3.7. Use '--feature-gates=TxnModeWriteWithSharedBuffer=true' instead.
```

Comment:
Refer to https://github.com/etcd-io/etcd/pull/19053#discussion_r1883688023

```suggestion
  --experimental-txn-mode-write-with-shared-buffer 'true'. Deprecated in v3.6 and will be decommissioned in v3.7. Use '--feature-gates=TxnModeWriteWithSharedBuffer=true' instead.
```